onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -2679,53 +2679,17 @@ It's an extension of Gelu. It takes the sum of input A and bias input B as the i
         updateOutputElemType(ctx, 0, ONNX_NAMESPACE::TensorProto::BOOL);
       });
 
-  static const char* OptionalEmpty_ver1_doc = R"DOC(
-      Construct an empty optional, with given element type.
-      )DOC";
-
-  ONNX_CONTRIB_OPERATOR_SCHEMA(OptionalEmpty)
-      .SetDomain(kMSDomain)
-      .SinceVersion(1)
-      .SetDoc(OptionalEmpty_ver1_doc)
-      .Attr("type", "Type of the element in the optional output", AttributeProto::TYPE_PROTO)
-      .Output(
-          0,
-          "output",
-          "An optional that contains no elements.",
-          "O")
-      .TypeConstraint(
-          "O",
-          {"optional(tensor(float))",
-           "optional(seq(tensor(float)))"},
-          "Constrains output type to all optional tensor or optional sequence types.")
-      .TypeAndShapeInferenceFunction([](InferenceContext& ctx) {
-        const auto* attr_proto = ctx.getAttribute("type");
-        if (attr_proto == nullptr)
-          fail_type_inference(
-                "Attribute type should be provided.");
-        if (!attr_proto->has_tp())
-          fail_type_inference(
-              "Attribute type should be of type TypeProto and specify a type.");
-        auto attr_tp = attr_proto->tp();
-        const size_t numOutputs = ctx.getNumOutputs();
-        if (numOutputs != 1) {
-          fail_type_inference("OptionalEmpty is expected to have 1 output.");
-        }
-        ctx.getOutputType(0)
-            ->mutable_optional_type()
-            ->mutable_elem_type()
-            ->CopyFrom(attr_tp);
-      });
-
   static const char* OptionalConstruct_ver1_doc = R"DOC(
-      Construct a optional type containing the 'input' element.
+      Construct an optional type containing either an empty optional of a certain type specified by the attribute, "
```

Comment:
Minor naming comment: I think we could just call this op "Optional" instead of "OptionalConstruct". Both shorter and more aligned with the common convention of type-name as a constructor.

Reply:
Sure. Updated.